sample/SampleAspNetCoreApp/appsettings.json

```diff
@@ -4,11 +4,12 @@
       "Default": "Warning"
     }
   },
-  "AllowedHosts": "*"//,
-  //Settings for the elastic APM Agent
-//  "ElasticApm":
-//    {
-//      "LogLevel":  "Debug",
-//      "ServerUrls":  "http://localhost:8200"
-//    }
+  "AllowedHosts": "*"
+  //,

```

Comment:
Does this change has any meaning or is it just accidental?
BTW I wasn't aware JSON supported comments - is it a non-standard extension?

Reply:
Yeah, this was intentional. I usually turn on debug log level during development, and it's very convenient to just uncomment this part for that. Also, I think it's nice to show this in the sample application for users, in case they also want to configure the agent. 

On the other hand you are right: JSON does not support comments, in fact Rider also gives a warning. Nevertheless, the effect of this is that the commented part will be ignored.

But now that you point this out I also think that this is not nice. 

I think it'd be better to have that config part there without commenting it out.

I see 2 options:
- Leave `"LogLevel":  "Debug",`: in this case when users start the sample app with the agent they will see lots of logs. - for me this is useful for development, for most of the people probably annoying.
- Change it to `"LogLevel":  "Error",`.

Reply:
Removed the comments, we simply add the part with the default settings. I think this still adds value:
- It helps users to discover how to configure the agent.
- It's easier to change it this way, e.g. for the loglevel we just need to change the value - instead of adding the whole thing.